ckanext/unaids/theme/templates/package/read_base.html

```diff
@@ -3,7 +3,7 @@
 {% block content_action %}
   {% if not is_activity_archive %}
     {% if h.check_access('package_update', {'id':pkg.id }) %}
-        <a class="btn btn-default" href="https://navigator.unaids.org" target="_blank">
+        <a class="btn btn-default" href="https://navigator.unaids.org/?datasetId={{ pkg.id }}" target="_blank">
```

Comment:
Can you include locale in the navigator URL based upon the selected ADR language?
This would require some mapping between ADR & Nav locale codes (e.g. pt_PT > pt)